### PR TITLE
Feature/issue934

### PR DESCRIPTION
**Pull request by chrisala**

### CODE REVIEW

**Review by temi — APPROVED** [review]

**Comment by chrisala** [comment]
Hi @temi I've added more changes to this PR to implement a delete and bulkUpdate.  Can you please review them?

File: grails-app/controllers/au/org/ala/ecodata/DataSetSummaryController.groovy [hunk 1]
```diff
@@ -27,6 +27,33 @@ class DataSetSummaryController {
...
+
+        if (!projectId) {
+            render status:  HttpStatus.SC_BAD_REQUEST, text: "projectId is required"
+            return
+        }
+
+        dataSets.each { Map dataSet ->
+            if (dataSet.projectId && dataSet.projectId != projectId) {
+                render status: HttpStatus.SC_BAD_REQUEST, text: "projectId must match the data set projectId"
+                return
```
Comment (temi) [inline grails-app/controllers/au/org/ala/ecodata/DataSetSummaryController.groovy:50]: just wondering if this return will exit controller action or just the each closure?

**Review by temi — COMMENTED** [review]
One comment about controller action

File: grails-app/controllers/au/org/ala/ecodata/DataSetSummaryController.groovy (hunk 1 above)
Comment (chrisala) [reply grails-app/controllers/au/org/ala/ecodata/DataSetSummaryController.groovy:50]: Thanks @temi, I've fixed this now.

**Review by temi — APPROVED** [review]

**Pull request merged**